event-app-api/models/user.js

```diff
@@ -23,7 +23,7 @@ export const User = sequelize.define("User", {
     type: DataTypes.STRING,
```

Comment:
[no action required now] Once you work on implementing the profile picture, you need to change the data type. It probably should be a blob